### PR TITLE
finger executable

### PR DESCRIPTION
**Pull request by omkar72**
helps to detect finger.exe execution. usually protocol blocked

### CODE REVIEW

**Comment by aw350m33d** [comment]
Hi @omkar72! Thanks a lot for your PRs! 👍🏻 

Apparently, you switched not from the `oscd` branch, but from the `master` branch, so you, along with your changes, push the existing ones in the main repository.
I ask you to correct this in all your PRs. 

It will help us a lot if the pull request contains only the rule that you propose to add or update in the repository. 
Thank you for understanding! 🤝

**Comment by e6e6e** [comment]
Hello @omkar72 !
Welcome and thanks for your participation!
Would you mind adding "oscd.community" to the author field in rule 07743f65-7ec9-404a-a519-913db7118a8d (and in other your PRs)?

File: rules/windows/process_creation/win_susp_finger.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+status: experimental
+description: finger.exe for data exfiltration or download file
+references:
+    - http://hyp3rlinx.altervista.org/advisories/Windows_TCPIP_Finger_Command_C2_Channel_and_Bypassing_Security_Software.txt
+author: omkar72, oscd.community
+date: 2020/10/11
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
```
Comment (e6e6e) [inline rules/windows/process_creation/win_susp_finger.yml:12]: I believe that you can add the following tags:
```
    - attack.command_and_control
    - attack.t1071
```
and delete the following tag:
```
    - attack.execution
```

**Comment by omkar72** [comment]
raising new pr

**Pull request closed without merging**